packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx

```diff
@@ -221,55 +210,19 @@ export default function MedicationOrderForm({ initialOrderBasketItem, onSign, on
             <>
               <Grid className={styles.gridRow}>
                 <Column md={4}>
-                  <ComboBox
+                  <NumberInput
                     id="doseSelection"
                     light={isTablet}
-                    items={dosageOptions}
-                    selectedItem={
-                      dosageOptions?.length
-                        ? {
-                            id: orderBasketItem.dosage ? `${orderBasketItem.dosage?.value}` : null,
-                            text: orderBasketItem.dosage ? `${orderBasketItem.dosage?.value}` : null,
-                          }
-                        : null
-                    }
-                    // @ts-ignore
                     placeholder={t('editDoseComboBoxPlaceholder', 'Dose')}
```

Comment:
Reckon we could omit the placeholder and just show a [default value](https://carbondesignsystem.com/components/number-input/usage/#default-values) instead (which we're already doing with the value prop). Additionally, I've noticed something subtle about how the NumberInput gets rendered:

<img width="1451" alt="Screenshot 2022-12-14 at 15 25 11" src="https://user-images.githubusercontent.com/8509731/207605769-1c6b8257-b518-4355-a863-e0a787837dee.png">

It doesn't quite fill the available vertical space. The culprit here is its `display: flex` property. You can apply this style override in the `gridRow` class to fix that:

```scss
.gridRow {
  @extend .row;
  padding: 0px;

  :global(.cds--number) {
    display: inline-block;
  }
}
```